public/locales/en/topics/ancestor-veneration.json

```diff
@@ -51,6 +51,6 @@
         { "src": "/images/4.png", "title": "making this the first true" },
         { "src": "/images/5.png", "title": "scrambled it to make a type" }
     ],
-    "galleryType": "mosaic",
+    "galleryType": "carousel",
```

Comment:
make sure to update the ja ones also, so there's less risk of things slipping through the gaps once we update with the real content copy

Reply:
Yessir o7